.github/workflows/build.yml

```diff
@@ -73,17 +84,18 @@ jobs:
 
             -
                 name: Run lint container (with Swift Mailer)
+                if: ${{ true == contains( matrix.symfony, '5.4.*' ) }}
                 run: |
                     rm -rf src/Bundle/test/var/cache
                     mv src/Bundle/test/config/packages/mailer.yaml mailer.yaml
                     composer remove symfony/mailer --dev
                     (cd src/Bundle/test && bin/console lint:container)
+                    mv mailer.yaml src/Bundle/test/config/packages/mailer.yaml 
```

Comment:
It would be great to unify these configuration shenanigans, as they look quite... messy :D On the other hand, they work for now so maybe let's improve them in a separate PR 🖖